google-home-community.groovy

```diff
@@ -2646,6 +2700,12 @@ private deleteDeviceTrait_OpenClose(deviceTrait) {
     app.removeSetting("${deviceTrait.name}.queryOnly")
 }
 
+@SuppressWarnings('UnusedPrivateMethod')
+private deleteDeviceTrait_CameraStream(deviceTrait) {
+    app.removeSetting("${deviceTrait.name}.GetCameraStream")
```

Comment:
I don't see where this `GetCameraStream` setting is set.  Can this line be removed?

Reply:
Agreed, will be removed.